textFilter.js

```diff
@@ -0,0 +1,63 @@
+// selects the elements in DOM
+const searchInput = document.querySelector('.input-search-keyword');
+const roomsUl = document.querySelector('.rooms');
```

Comment:
By using these global variables, you are setting yourself up for bugs. Instead of using global variables that all the functions access from the global scope, query the elements in one place and then pass them as arguments into the functions that need them.